package.json

```diff
@@ -572,13 +575,13 @@
       ],
       "editor/title/run": [
         {
-          "when": "editorFocus && editorLangId == sas && !SAS.hideRunMenuItem",
+          "when": "editorLangId == sas && !SAS.hideRunMenuItem",
           "command": "SAS.runSelected"
```

Comment:
replace all 'SAS.hideRunMenuItem' with SAS.running ?

Reply:
We're disabling instead of hiding the icon for running. Hiding is still needed when execution is completely not supported (pure web).